homeTask7/src/main/java/org/example/servlets/ListServlet.java

```diff
@@ -0,0 +1,67 @@
+package org.example.servlets;
+
+import org.example.driver.MySQLDriverManager;
+import org.example.entity.Table;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static java.lang.System.out;
+
+public class ListServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        MySQLDriverManager driverManager = MySQLDriverManager.getInstance();
+        Connection connection;
+        PreparedStatement preparedStatement = null;
+        ResultSet resultSet = null;
+
+        try {
+            connection = driverManager.getConnection();
+            connection.setAutoCommit(false);
```

Comment:
There is no need to use `setAutoCommit(false)` when you use `SELECT` queries. It's needed only when we modify database data